scss/titles.scss

```diff
@@ -0,0 +1,18 @@
+.title-container {  
+    height: 90px;
+    width: 100px;
+
+    h2 {
```

Comment:
Create style in classes for each tag. It is the best way to reuse them along all your html.
For example if you want to reuse this style in another side that is not nested in `.title-container` you will not be able to use it.
`So i will recommend to create another separated style like 'tech-title' or 'main-title'`